tests/Multi/SearchTest.php

```diff
@@ -292,6 +292,7 @@ public function testSearchWithError(): void
 
         $multiSearch->addSearch($searchBad);
 
+        $this->markTestSkipped('Elastica\Exception\ResponseException: [size] parameter cannot be negative, found [-2]');
```

Comment:
In this test it seems we cause an error on purpose by passing in -2 (same below). No ES does proper validation and returns an error already here. Easiest change would be to just adjust the test and check for the error here or even remove the test.